src/components/common/SideMenu.js

```diff
@@ -119,6 +135,20 @@ class SideMenu extends React.Component {
     dispatchSetIsPaused(!isPaused);
   };
 
+  reset = () => {
+    const { dispatchResetSettings } = this.props;
+    dispatchResetSettings();
+  };
+
+  onScaleUnitToggle = () => {
+    const { dispatchSetScaleUnit, showKelvinScale } = this.props;
+    if (showKelvinScale) {
```

Comment:
Shouldn't this be the opposite? If it's not the verb but the boolean, then maybe rename `showKelvinScale: currentlyShowingKelvinScale` or something like that.